lib/verk/workers_manager.ex

```diff
@@ -99,7 +99,15 @@ defmodule Verk.WorkersManager do
     if workers != 0 do
       case QueueManager.dequeue(state.queue_manager_name, workers) do
         jobs when is_list(jobs) ->
-          for job <- jobs, do: job |> Job.decode! |> start_job(state)
+          Enum.map(jobs, fn job ->
+            try do
+              job |> Job.decode! |> start_job(state)
```

Comment:
Yeah I think we should use `Job.decode` -> `Poison.decode` and treat the error case 👍